app/views/comments/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="form-container">
+  <h1>Create New Comment</h1>
+  <%= form_with model: [@user, @post, @comment] do |f| %>
+    <h3>Text:</h3>
+    <%= f.text_area :text, placeholder: "Write your text...", required: true, class: "text-area" %>
+    <div class="submit-button">
+      <%= f.submit "Submit", class: "button submit" %>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Kindly make sure this error is solved when a comment is created, check the redirection when a comment is created.

![image](https://user-images.githubusercontent.com/65068771/187003187-ce51eb7f-995c-49a7-80a7-a34331397dad.png)